appendix.md

```diff
@@ -27,7 +27,7 @@ Type in:
 $ echo test | faas-cli invoke non-existing-function
 ```
 
-Now give it a few seconds and check what you see on the UI. There should be a 500 error for the function name `non-existing-function.
+Now give it a few seconds and check what you see on the UI. There should be a 404 error for the function name `non-existing-function`.
```

Comment:
This was actually 404 not 500.

Reply:
It could be different depending on a number of factors. Perhaps we could say a HTTP error code?